packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -101,46 +102,42 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
   const { USDC } = terms.brands;
   const { withdrawToSeat } = tools.zoeTools;
   const { baggage, chainHub, orchestrateAll, vowTools } = tools;
-  const makeSettler = prepareSettler(zone, {
-    statusManager,
-    USDC,
-    withdrawToSeat,
-    feeConfig,
-    vowTools: tools.vowTools,
-    zcf,
-    chainHub,
-  });
 
   const { localTransfer } = makeZoeTools(zcf, vowTools);
-  const makeAdvancer = prepareAdvancer(zone, {
-    chainHub,
-    feeConfig,
-    localTransfer,
-    usdc: harden({
-      brand: terms.brands.USDC,
-      denom: terms.usdcDenom,
+
+  const { make } = organizeMakers({
+    ...orchestrateAll(flows, {}),
+    makeSettler: prepareSettler(zone, {
```

Comment:
I thought about a record like this, but sometimes you have to pass `makeA` into `prepareB`.

Reply:
oh good point. That thought crossed my mind while I was refactoring but it consolidating like this was too delicious